util/packaging/aur/chapel-git/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+
+# Maintainer: Gabriel Brown <gabriel.h.brown@gmail.com>
+
+#_gituser="chapel-lang"
+_gituser="mppf" #temp: until PR #19913 is merged
+_gitname="chapel"
+
+pkgname=${_gitname}-git
+_basepkgver=1.26
+pkgver=1.26.cc0241ef9c
+pkgrel=1
+pkgdesc="Programming language designed for productive parallel computing at scale"
+url="https://chapel-lang.org/"
+arch=('x86_64' 'arm')
+license=('Apache')
+depends=('python' 'perl' 'llvm' 'clang')
+makedepends=('git' 'cmake')
+source=("git+https://github.com/${_gituser}/${_gitname}.git")
```

Comment:
I don't think this was using my branch, yet. However I am expecting to merge PR #19913 on Monday and my branch is working in my own testing.

Reply:
(my `main` branch usually just tracks upstream/main but I just pushed a merge of PR #19913 in to it, FWIW).

Reply:
Right, I check out your `less-la` branch on line 28, if that's what you are referring to. Hopefully that's doing what I think it is, butI didn't have any issues.

Reply:
Ah, gotcha, thanks, I missed that.